examples/dt/uni02beta/control-plane/service-values.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinder:
+    extraMounts:
+      - extraVol:
+          - propagation:
+              - CinderVolume
+            extraVolType: Undefined
+            volumes:
+              - name: nfs-shares
+                projected:
+                  sources:
+                    - secret:
+                        name: cinder-volume-ontap-shares-secrets
+            mounts:
+              - name: nfs-shares
+                mountPath: /etc/cinder/nfs_shares.d
+                readOnly: true
+
+  cinderVolumes:
+    ontap-nfs:
+      customServiceConfig: |
+        [ontap]
+        volume_backend_name=ontap
+        volume_driver=cinder.volume.drivers.netapp.common.NetAppDriver
+        netapp_server_hostname=_replaced_  # CHANGEME
+        netapp_server_port=80
+        netapp_storage_protocol=nfs
+        netapp_storage_family=ontap_cluster
+        nfs_shares_config=/etc/cinder/nfs_shares.d/shares-config
+    networkAttachments:
+      - storage
+    customServiceConfigSecrets:
+      - cinder-volume-ontap-secrets
+
+  ontap-cinder-secrets.conf: |
+    [ontap]
+    netapp_login = _replaced_
+    netapp_password = _replaced_
+    netapp_vserver = _replaced_
+    nas_host = _replaced_
+    nas_shares_path = _replaced_
+
+  shares-config: |
+    192.168.1.1:/nfs/test # CHANGEME
+    192.168.1.1:/nfs/test2 # CHANGEME
+
+  manila:
+    enabled: true
+    manilaAPI:
+      customServiceConfig: |
+        [DEFAULT]
+        debug = true
+        enabled_share_protocols=cifs
+      replicas: 1
+    manilaScheduler:
+      replicas: 1
+    manilaShares:
+      share1:
+        networkAttachments:
+          - storage
+        customServiceConfig: |
+          [DEFAULT]
+          debug = true
+          enabled_share_backends=netapp
+          [netapp]
+          driver_handles_share_servers=False
+          share_backend_name=netapp
+          share_driver=manila.share.drivers.netapp.common.NetAppDriver
+          netapp_storage_family=ontap_cluster
+          netapp_transport_type=http
+        customServiceConfigSecrets:
+          - osp-secret-manila-netapp
+        replicas: 1
+
+    netapp-secrets.conf: |
+      [netapp]
+      netapp_server_hostname = _replaced_
+      netapp_login = _replaced_
+      netapp_password = _replaced_
+      netapp_vserver = _replaced_
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:file
+      [glance_store]
+      default_backend = default_backend
+      [default_backend]
+      filesystem_store_datadir = /var/lib/glance/images/
+    databaseInstance: openstack
+    glanceAPIs:
+      default:
+        preserveJobs: false
+        replicas: 1
+        type: single
+    extraMounts:
```

Comment:
+1 to have `extraMounts` local to each service. This is a good practice and I think it can help to define per-component things without the need of build a verbose struct at the top-level.